hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -127,9 +127,9 @@ public void migrate(@NonNull final MigrationContext ctx) {
             if (nodeDetailMap != null) {
                 nodeDetail = nodeDetailMap.get(nodeInfo.nodeId());
                 if (nodeDetail != null) {
-                    nodeBuilder
-                            .serviceEndpoint(nodeDetail.serviceEndpoint())
-                            .grpcCertificateHash(nodeDetail.nodeCertHash());
+                    final Bytes hashBytes =
+                            Bytes.fromHex(nodeDetail.nodeCertHash().asUtf8String());
```

Comment:
This converting Bytes to Bytes is not necessary and not correct. Node's grpcCertificateHash is the same Bytes as NodeAddress.nodeCertHash.

Reply:
You are incorrect here, as discussed in the meeting.  The team agreed to this change.